modules/badge/react/README.md

```diff
@@ -0,0 +1,76 @@
+# Canvas Kit React Badge
+
+CountBadge provides a quantity-based summary with dynamic values.
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-react
+```
+
+or
+
+```sh
+yarn add @workday/canvas-kit-react-badge
+```
+
+## Accessibility
+
+If the count is live-updated (as in the case of notifications) and not static, please add
+`aria-live="polite"` to inform screen readers to announce updated information. When this attribute
+is set the screen reader will announce changes when the user is idle.
+
+```jsx
+<Badge count={3} aria-live="polite" aria-label="3 direct reports" />
+```
+
+## Usage
+
+```tsx
+import * as React from 'react';
+import Badge from '@workday/canvas-kit-react-badge';
```

Comment:
Just realized that docs have to change with our new component naming and named exports